server/app/services/export/QuestionJsonSampler.java

```diff
@@ -37,10 +35,11 @@
  */
 public interface QuestionJsonSampler<Q extends Question> {
 
-  default CfJsonDocumentContext getSampleJson(QuestionDefinition questionDefinition) {
+  default ImmutableMap<Path, Optional<?>> getSampleJsonEntries(
+      QuestionDefinition questionDefinition) {
     if (questionDefinition.getEnumeratorId().isPresent()) {
```

Comment:
`getSampleJson` is renamed to `getSampleJsonEntries` because it now returns an `ImmutableMap`; `QuestionJsonSampler` was only being used to merge the returned `CfJsonDocumentContext`s before, so this is more robust.